src/utils/gas.ts

```diff
@@ -117,6 +117,12 @@ export function updateGasPricing(
       gasPricing.maxFeePerGas = gasPricing.nextBlockFee!.add(gasPricing.nextPriorityFee!)
       gasPricing.gasPrice = gasPricing.maxFeePerGas
     }
+  } else {
+    if (gasPricing.gasPrice === null) {
```

Comment:
When you make the update with more verbose logging can you also just add comment that this is for EIP1559

Reply:
Shouldn't get triggered on any network except those that don't support EIP1559